server/etcdserver/server.go

```diff
@@ -2028,12 +2032,14 @@ func (s *EtcdServer) apply(
 
 		case raftpb.EntryConfChange:
 			// set the consistent index of current executing entry
+			shouldApplyV3 := false
 			if e.Index > s.consistIndex.ConsistentIndex() {
 				s.consistIndex.SetConsistentIndex(e.Index)
+				shouldApplyV3 = true
```

Comment:
Nit: Could we leave a comment to explain why we change the applyV3 to true, might be helpful down the road.

Reply:
Done.